src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -5384,9 +5420,16 @@ private void ParseTypeArgumentList(out SyntaxToken open, SeparatedSyntaxListBuil
                 {
                     break;
                 }
-                else if (this.CurrentToken.Kind == SyntaxKind.CommaToken || this.IsPossibleType())
+                else if (this.CurrentToken.Kind == SyntaxKind.CommaToken ||
+                         this.IsPossibleType())
```

Comment:
This is code style change. Wouldn't it be more appropriate to move that to separate commit?